capabilities.go

```diff
@@ -139,5 +151,54 @@ func (c *Client) GetCapabilities(target string, port int) (response *Capabilitie
 		err = fmt.Errorf("missing %s version", DefaultServiceName)
 	}
 
+	// Parse PIKE capability
+	if pike, ok := response.Capabilities["935478af7bf2"].(map[string]interface{}); ok {
+		pikeCap := &PikeCapability{
+			Invite:  pike["invite"].(string),
+			Outputs: pike["outputs"].(string),
+		}
+		response.Pike = pikeCap
+	}
+
 	return
 }
+
+// ExtractPikeOutputsURL extracts the outputs URL from the PIKE capability
+func (c *CapabilitiesPayload) ExtractPikeOutputsURL() string {
+	if c.Pike != nil {
+		return c.Pike.Outputs
+	}
+	return ""
+}
+
+// GetOutputsTemplate calls the PIKE capability outputs subcapability
+func (c *Client) GetOutputsTemplate(pikeURL string) (response *PikeOutputs, err error) {
+	var resp StandardResponse
+	if resp, err = c.getRequest(pikeURL); err != nil {
```

Comment:
You should use c.postRequest with some parameters (`paymail.PikePaymentOutputsPayload`)  otherwise it won't work. You can see how it was done in `p2p_payment_destination.go` `GetP2PPaymentDestination()`